numpy/ma/extras.py

```diff
@@ -1377,9 +1376,9 @@ def cov(x, y=None, rowvar=True, bias=False, allow_masked=True, ddof=None):
     return result
 
 
-def corrcoef(x, y=None, rowvar=True, bias=False, allow_masked=True, ddof=None):
+def corrcoef(x, y=None, rowvar=True, bias=np.NoValue, *args, **kwargs):
```

Comment:
Don't we lose the the ability to check for a positional argument for `allow_masked` in that case?